Will Heikoff Documentation

```diff
@@ -320,4 +320,64 @@ The assumption that the Zip Code Business Pattern Data is based off of USPS Zip
+
+8/2/18
+Changed CA_WAC_Transformation.R file to open file directly from the gz file and format table without the use of excel at all
+Realized error with K-function Local_3Stage.py in that it only creates a …_points.shp and not a …_buffers.shp. This means that it creates a shapefile with the lab points but not the clusters
+“GDAL data files not located, GDAL_DATA not set” and “PROJ data files not located, PROJ_LIB not set” errors seen in the K-function Local_3Stage.py log
+Troubleshooting to find solution; may be due to the ZIPS.shp file perhaps being a based off of ZCTAs and not USPS Zip Codes
+8/3/18
+Dowloaded California’s 2000 ZCTA Profile of Selected Economic Characteristics from https://factfinder.census.gov/faces/nav/jsf/pages/searchresults.xhtml?refresh=t# with the Product Type Filter: Data Profile, Year: 2000, 5-Digit ZCTA: All 5-Digit ZIP Code Tabulation Areas fully within/partially within California, saved as DEC_00_SF3_DP3_with_ann.csv
+Wrote script in R to import and format the DEC_00_SF3_DP3_with_ann.csv file with only the ZCTA ID and manufacturing employment and export the file as ZCTA_Manufa.dbf, saved in the Complete Zip Code Industry Detail File. Script saved as ZCTA_Manufa_Trans.R in the same folder
+Created CAL_ZCTA_EMP_2.shp file which contains a join of the ZCTA_Manufa.dbf file and the ZIPS.shp, saved in the Complete Zip Code Industry Detail File, changed CRS to USA_Contiguous_Albers_Equal_Area_Conic Coordinate System
+Working with K-function Local_3Stage.py program - Entered “G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Complete Zip Code Industry Detail File\CAL_ZCTA_EMP_2.shp” into line 37 to set the zips file
+Entered “Number__Em” and “Id2” onto line 40 to set the zip features
+Entered “Number__Em” on line 44 to set the Reference Measure
+Entered “Id2” on line 47 to set the ID Field
+Running program with new Zip Code File for two simulations
+Simulations completed in 3.35 minutes
+Received same “GDAL data files not located, GDAL_DATA not set” and “PROJ data files not located, PROJ_LIB not set” errors, and …_buffers.shp file not created
+Trying to set GDAL by entering “conda create -n GDAL python=3.6 gdal -c conda-forge” into Anaconda Prompt
+	Installation successful; after running the program again, the errors still occur and still no …_buffers.shp built
+
+8/6/18
+Adjusted CA_WAC_Transformation.R file to format the Block ID to add the necessary 0 at the beginning of the ID directly in R rather than having to do it in ArcMap
+Updated ZIPs_Tot_Emp.R file to extract only California Zip Codes from the file
+2000 ZCTA Manufa. Data has 198 Zips with no Manufa. Employment, 2000 ZBP Manufa. Data has 0 Zips with no Manufa. Employment, 1998 ZBP Manufa. Data has 0 Zips with no Manufa. Employment
+Earliest ZCTA Shapefiles available are 2007
+Downloaded 2000 ZCTA Cartographic Boundary Files for California ftp://ftp2.census.gov/geo/tiger/PREVGENZ/zt/z500shp/, titled zt06_d00.shp, saved on the desktop
```

Comment:
We have something better than this, right? So we can delete this and make a note that we didn't end up using it?